conn2res/task.py

```diff
@@ -9,7 +9,7 @@
 import numpy as np
 import pandas as pd
 import scipy as sp
-import mdp
+# import mdp
```

Comment:
I commented it out as it threw and error and I realized that the mdp module is not used anyway at the moment.